src/nupic/regions/TestNode.cpp

```diff
@@ -143,6 +143,8 @@ Spec *TestNode::createSpec() {
   ns->description = "TestNode. Used as a plain simple plugin Region for unit tests only. "
       "This is not useful for any real applicaton.";
 
+  ns->singleNodeOnly = false; // this means we allow dimensions;
```

Comment:
drop singleNodeOnly altogether?

Reply:
> drop singleNodeOnly altogether?

Yes,  But hanging on to it until we are sure this is what we want.  See discussion on dimensions.